keepassxc-browser/css/keepassxc-browser.css

```diff
@@ -0,0 +1,14 @@
+.cip-genpw-icon {
+    position: absolute;
+    cursor: pointer;
+}
+
+.cip-genpw-icon.key {
+    background: url('chrome-extension://__MSG_@@extension_id__/icons/key.png') right no-repeat;
+    background-size: contain;
+}
+
+.cip-genpw-icon.key-moz {
+    background: url('moz-extension://__MSG_@@extension_id__/icons/key.png') right no-repeat;
```

Comment:
Or better like that, so it's still an absolute path, but without that ugly "protocol":

```suggestion
    background: url('/icons/key.png') right no-repeat;
```

Reply:
That just doesn't work. Even adding the style manually to the JS code the icon doesn't work properly without this kind of path. You can try and verify it yourself.

Reply:
Ah, it is in a content script.